wdl-engine/src/inputs.rs

```diff
@@ -753,13 +815,12 @@ impl Inputs {
     ) -> Result<(String, Self)> {
         let mut inputs = TaskInputs::default();
         for (key, value) in object {
-            let value = serde_json::from_value(value)
-                .with_context(|| format!("invalid input key `{key}`"))?;
+            let value = serde_json::from_value(value).context("invalid input key")?;
```

Comment:
We should revert the change to this line. When I mentioned using `context` instead of `with_context`, it only applied to the case where we are not formatting a string. This message formats with the invalid input key, which is useful for users.